kork-core/src/main/java/com/netflix/spinnaker/kork/archaius/ArchaiusConfiguration.java

```diff
@@ -37,6 +38,7 @@
 import org.springframework.core.io.support.ResourcePropertySource;
 
 @Configuration
+@ConditionalOnExpression("${archaius.enabled:true}")
```

Comment:
```suggestion
@ConditionalOnProperty(value = "archaius.enabled", matchIfMissing = true)
```

I'd prefer an explicit property reference to an expression here

Reply:
Save a Developer; Avoid SpEL.